internify/src/components/molecules/Notes.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+import { Chip } from "@material-ui/core";
+import "./styles/Notes.css";
+
+
+const Notes = ({canadaMatches, ubcMatches, totalMatches, totalStudents}) => {
+
+    const msg1 = `${totalMatches} out of ${totalStudents} matches in our DB`
+    const msg2 = `${ubcMatches} matches with UBC students`
+    const msg3 = `${canadaMatches} total matches in Canada`
```

Comment:
Are these messages fixed? I know you're following designs from the Landing page but they aren't fixed and final lol. We can accept this for now but in the future we need to talk about what kinds of messages we wanna be showing.